circus-train-distcp-copier/src/main/java/com/hotels/bdp/circustrain/distcpcopier/DistCpCopier.java

```diff
@@ -56,6 +56,27 @@ public Job exec(Configuration conf, DistCpOptions options) throws Exception {
 
   private static final Logger LOG = LoggerFactory.getLogger(DistCpCopier.class);
 
+  private static class JobCounter implements Gauge<Long> {
+
+    private final Job job;
+    private final String counter;
+
+    public JobCounter(Job job, String counter) {
+      this.job = job;
+      this.counter = counter;
+    }
+
+    @Override
+    public Long getValue() {
+      try {
+        return job.getCounters().findCounter(FileSystemCounter.class.getName(), counter).getValue();
```

Comment:
This will only find Counters with a FileSystemCounter group name so a more accurate name for the class would be `FileSystemCounterGauge`